definitions/types/gcp-grn.json

```diff
@@ -4,7 +4,11 @@
   "title": "GCP Resource Name (GRN)",
   "description": "GCP Resource Name (GRN)",
   "examples": [
-    "projects/my-project/locations/us-central1/instances/my-instance"
-  ]
+    "projects/<my-project>/locations/<location>/instances/<my-instance>",
+    "projects/<my-project>/locations/<location>/clusters/<my-cluster>"
+  ],
+  "pattern": "^projects\/[a-z0-9-]*\/locations\/[a-z0-9-]*\/[a-z0-9-]*\/.*$",
```

Comment:
Can GRN's ever by longer that 6 blocks?
like `projects/.../locations/.../clusters/my-cluster/node/mynodegroup`?

Reply:
That's a great question, I think they all follow this but nodegroup is worth checking.

I know this will break the Firestore bundle, we're passing back a grn of the id of the Firestore database. It should be a built up full-path, I'll add a ticket.

Reply:
Okay, this is the grn of a GKE node pool.

`projects/md-wbeebe-0504/zones/us-west2-a/instances/gke-testpol-test-gke-rf7n-small-pool-7e872d0d-esci`

So the locations regex doesn't hold up here. I'll make that `[a-z]+`